api-spec.yaml

```diff
@@ -0,0 +1,151 @@
+consumes:
+- application/json
+info:
+  description: Column - Ansible as a service
+  title: Column
+  version: 0.0.1
```

Comment:
what version is this?  shouldn't it match column version? if so, use bumpversion mechanism